src/uct/api/uct.h

```diff
@@ -426,6 +418,32 @@ typedef enum uct_atomic_op {
  */
 
 
+/**
+ * @defgroup UCT_RESOURCE_IFACE_EVENT_CAP   UCT interface event operations and capabilities
```

Comment:
UCT interface for asynchronous events capabilities

Reply:
done